glean-core/python/glean/util.py

```diff
@@ -22,3 +23,10 @@ def get_locale_tag() -> str:
     value = value.replace("_", "-")
 
     return value
+
+
+def time_ms() -> int:
```

Comment:
I bet there's no concept of `uint`, right?

Reply:
That's correct: Python 3 only has a single `int` type.